### ISSUE TITLE
img missing alt text

### ISSUE THREAD

**Issue by discospiff** [Jose43210/IS7012#10]
Use the alt attribute on the img tag to describe the image to a user who can't see it. 

https://github.com/Jose43210/IS7012/blob/e5f25256422d9f25acacbca91bfcc7a293fab091/esquivjs-about-me.html#L12
